tools/wptrunner/wptrunner/executors/executorservodriver.py

```diff
@@ -235,8 +235,8 @@ def __init__(self, browser, server_config, timeout_multiplier=1,
                                                capabilities=capabilities)
         self.implementation = RefTestImplementation(self)
         self.timeout = None
-        with open(os.path.join(here, "reftest-wait_webdriver.js")) as f:
```

Comment:
Does Servo still use this executor? If so, is reftest-wait completely broken in Servo now?

Reply:
I'm not sure that servo ever switched to this executor, but @jdm would know more.